packages/data-store/src/types/digitalCredential/digitalCredential.ts

```diff
@@ -0,0 +1,46 @@
+export type NonPersistedDigitalCredential = Omit<DigitalCredential, 'id'>
+
+export type DigitalCredential = {
+  id: string
+  credentialType: CredentialType
+  documentFormat: CredentialDocumentFormat
+  raw: string
```

Comment:
Suggest to rename this one to rawDocument, as we also already have uniformDocument

Reply:
renamed to `rawDocument`